cases-SEP31/customer.test.js

```diff
@@ -116,25 +116,61 @@ describe("/customer", () => {
         jwt,
         false,
         fieldsRequired,
+      ));
+      if (customerValues["first_name"]) {
+        customerValues["first_name"] += "test";
+      } else {
+        // This test is useless if the anchor doesn't require first names.
+        // I assume every anchor would want this.
+        return;
```

Comment:
Instead of returning would it be best to simply make the `customerValues["first_name"]` value == "test"?